index.html

```diff
@@ -425,11 +425,12 @@ <h2>LDP-NRs</h2>
         <h2>HTTP PUT</h2>
         <p>
           When accepting a <code>PUT</code> request against an existing resource, an HTTP <code>Link: rel="type"</code>
-          header MAY be included. If the type in the Link header is a subtype of a current type of the resource, and
-          has an interaction model assigned to it by [[!LDP]], then the resource MUST be assigned the new type and the
-          interaction model of the resource MUST be changed to the interaction model assigned to the new type by
-          [[!LDP]]. If that type is a value in the LDP namespace and is not either a current type of the resource or a
-          subtype of a current type of the resource, the request MUST be rejected with a 409 (Conflict) response.
+          header MAY be included. If the type in the Link header is a subtype of a current type of the resource, is in
+          the [[!LDP]] namespace, and has an interaction model assigned to it by [[!LDP]], then the resource MUST be
```

Comment:
would LDP ever assign an interaction model to something not in the LDP namespace? Could we get rid of the _"is in the LDP namespace"_ clause?

Reply:
I think the intention here is to only allow changing interaction models within the LDP-type class hierarchy.
We want to avoid making any assertions about changing the RDF-type of a resource based on its sub-type relationship to other arbitrary RDF-types.
It sounds like the wording could be updated to make that more clear.

Reply:
It's already a mouthful.  I think it's time to break this monster sentence apart, if possible.